project/Build.scala

```diff
@@ -805,6 +805,74 @@ object Build {
       javaOptions := (javaOptions in `scala3-compiler-bootstrapped`).value
     )
 
+  /** Scala library compiled by dotty using the latest published sources of the library */
+  lazy val `scala3-scala2-library` = project.in(file("scala3-scala2-library")).
```

Comment:
I'm not a fan of that name, I suggest `stdlib-booststrapped` or `stdlib-recompiled` maybe.